DiscordBee.cs

```diff
@@ -128,6 +128,8 @@ public void ReceiveNotification(string sourceFileUrl, NotificationType type)
         case NotificationType.TrackChanged:
         case NotificationType.PlayStateChanged:
           UpdateDiscordPresence(_mbApiInterface.Player_GetPlayState());
+          if (type == NotificationType.TrackChanged)
```

Comment:
Why is this needed? isn't that covered by `PlayingTracksChanged` already?

Reply:
Nope, that event is not fired when the current track ends and the next one begins.